lib/findMatchingFiles.js

```diff
@@ -49,6 +51,26 @@ function findMatchingFilesWithNode(
   });
 }
 
+function findMatchingFilesWithWatchman(
+  lookupPath: string,
+  validFilesRegex: string
+): Promise<Array<string>> {
+  const tester = new RegExp(validFilesRegex, 'i');
```

Comment:
I think `validFilesRegex` should be `validFilesPattern`, and for this `RegExp` instance to be `validFileRegex`